src/App.vue

```diff
@@ -100,6 +164,18 @@ export default {
       }
       return "";
     },
+    logout: function () {
```

Comment:
From the content of this logout function, am I to assume that whatever credentials the App obtains via OIDC authentication, cannot be properly revoked upon logging out? Since all the logout function does is erase the cookie?

Reply:
Added revocation request in the latest commit.